src/node.rs

```diff
@@ -588,10 +588,47 @@ impl EthNamespaceT for InMemoryNode {
                 U64::from(0)
             };
 
+            let logs: Vec<Log> = info
+                .result
+                .result
+                .logs
+                .events
+                .iter()
+                .map(|log| {
+                    let address = log.address;
+                    let topics = &log.indexed_topics;
+                    let data = log.value.clone();
+                    let block_hash = Some(hash);
+                    let block_number = Some(U64::from(info.miniblock_number));
+                    let l1_batch_number = Some(U64::from(info.batch_number as u64));
+                    let transaction_hash = Some(hash);
```

Comment:
I'm guessing having the `block_hash` and `transaction_hash` be the same value shouldn't be a problem if we are enforcing 1 transaction per block?

Reply:
Yes, and this is definitely a temporary solution as we continue to implement the remaining parts of the api.